bagitobjecttransfer/recordtransfer/admin.py

```diff
@@ -136,25 +137,18 @@ def save_model(self, request, obj, form, change):
     def _get_changed_message(changed_data: list, user: User):
         """ Generate a list of changed status message for certain account details. """
         message_list = list()
-        for item in changed_data:
-            message = None
-            change = None
-            if item == "is_superuser":
-                message = "Superuser privileges"
-                change = user.is_superuser
-            elif item == "is_staff":
-                message = "Staff privileges"
-                change = user.is_staff
-            if message is not None:
-                message += " have been {} your account".format("added to" if change else "removed from")
-                message_list.append(message)
+        if "is_superuser" in changed_data:
+            if user.is_superuser:
+                message_list.append("Superuser privileges have been added to your account.")
+            else:
+                message_list.append("Superuser privileges have been removed from your account.")
+        if "is_staff" in changed_data:
+            if user.is_staff:
+                message_list.append("Staff privileges have been added to your account.")
+            else:
+                message_list.append("Staff privileges have been removed from your account.")
```

Comment:
These should use gettext(), but I like this new structure, it's more readable than before.